src/routes/productRouter.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const productRouter = express.Router();
+
+const productController = require('../controllers/productController');
+const { checkAuth } = require('../utils/checkAuth');
+
+productRouter.get('/', productController.getAllProducts);
```

Comment:
```suggestion
productRouter.get('', productController.getAllProducts);
```
/를 작성하려면 다른 주소들도 마찬가지로 맨 뒤에 /를 붙여야 통일성있다고 볼 수 있습니다.

하지만 `/` 자체를 적을 필요가 없기 때문에 아예 전부 적지 않는 것입니다.

Reply:
넵 알겠습니다!